components/icu/Cargo.toml

```diff
@@ -4,7 +4,7 @@
 
 [package]
 name = "icu"
-description = "International Components for Unicode"
+description = "ICU4X Project: Unicode Internationalization Library for Rust"
```

Comment:
it would be useful to expand the acronym somewhere

Reply:
The acronym doesn't expand; it is "International Components for Unicode for X" which doesn't make sense. We're not an X (Twitter) specific project.

Reply:
ICU expands

Reply:
Which is the crate's name